activejob/test/cases/test_helper_test.rb

```diff
@@ -555,6 +556,12 @@ def test_assert_enqueued_with_with_no_block_with_at_option
     assert_enqueued_with(job: HelloJob, at: Date.tomorrow.noon)
   end
 
+  def test_assert_enqueued_with_with_hash_arg
```

Comment:
Please let me know if there's something else I should test instead(or at some other place).